client/src/components/regform/regform.js

```diff
@@ -1,5 +1,216 @@
 import React, { Component } from "react";
+import cookie from 'react-cookie'
+import './styles'
 
-export const Regform = () => {
-  return <div>Hello from Regform</div>;
-};
+class Regform extends Component {
+  constructor(props) {
```

Comment:
You don't need constructors in the recent versions of React. Try this:
```
state = {
 ...
}
```
and delete the super().

Reply:
I will add it to the refactor list. Thanks.